src/js/msp/MSPHelper.js

```diff
@@ -1142,6 +1142,12 @@ MspHelper.prototype.process_data = function(dataHandler) {
 
                                         if(semver.gte(CONFIG.apiVersion, "1.42.0")) {
                                             ADVANCED_TUNING.itermRelaxCutoff = data.readU8();
+											
+                                            if(semver.gte(CONFIG.apiVersion, "1.43.0")) {
```

Comment:
This has to be 1.44.0, as this will only go into configurator after 4.2 has been released.